lms/djangoapps/discussion_api/api.py

```diff
@@ -263,20 +279,38 @@ def get_thread_list(request, course_key, page, page_size, topic_id_list=None, te
     if exclusive_param_count > 1:  # pragma: no cover
         raise ValueError("More than one mutually exclusive param passed to get_thread_list")
 
+    # These should already be validated by the django forms.
```

Comment:
I think this comment is rather confusing. They are only validated by the Django forms when the view calls the api method, right? It's not true for any direct calls to the api. I'd recommend removing the comment, or clarifying that when the view method calls the API method, the django forms will already have validated these values.